src/lib/languages.json

```diff
@@ -0,0 +1,433 @@
+{
+  "af-NA": "Afrikaans (Namibia)",
```

Comment:
I think it'd be more friendly to display the languages in their local name, rather than english for all the things.

<img width="639" alt="screenshot 2018-11-02 at 12 09 34" src="https://user-images.githubusercontent.com/58871/47915046-8bb41e80-de99-11e8-958b-f388f1a9044d.png">

Reply:
@lidel have you got any sources for languages in their local language. How do we deal with the multiple Chinese varients? I'm stuggling to find a simple process for this. I have pinged transifex.

Reply:
More here https://github.com/ipfs/i18n/issues/4

Reply:
I was about to link https://github.com/mozilla/language-mapping-list/blob/master/language-mapping-list.js but you already found it :) (`nativeName` is what we want to show to end user)